schemas/data/page-views.schema.json

```diff
@@ -45,5 +45,5 @@
     }
   ],
   "required": ["@id", "schema:name", "xdm:measurement", "xdm:unit"],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
👍

Reply:
👎

- It uses schema:name instead of xdm:name